src/Table/Table.js

```diff
@@ -383,7 +383,8 @@ export default class Table extends React.Component {
 
     // Use scroll table on first render to check if we need to scroll
     // and if content is bigger than its container
-    if (props.useFlex || itemHeight === 0 || itemListHeight > containerHeight) {
+    if ((props.useFlex || itemHeight === 0 || itemListHeight > containerHeight)
+      && props.useScrollTable === true) {
```

Comment:
can we just do `&& props.useScrollTable` without the === true? after that feel free to merge